app/templates/components/send-message.hbs

```diff
@@ -1,13 +1,18 @@
-<form class="form-group" {{action 'createMessage' on='submit'}}>
-  <fieldset>Message must be under 1000 characters.</fieldset>
-  {{textarea
-    required="true"
-    maxlength="1000"
-    cols="40"
-    rows="6"
-    class="form-control"
-    value=newMessage.body
-    placeholder='Type Message Here'
-  }}
-  <button type=submit class="btn btn-primary btn-block">Send</button>
-</form>
+{{paper-divider}}
+{{#paper-form onSubmit="createMessage"}}
+  <H3>Send a message here!</H3>
+  <p>Message must be under 750 characters</p>
```

Comment:
For the future it would be cool to use `newMessage.body.length / 750` instead so the user can see how many characters are remaining. It is also more elegant and will have a nicer tone.